Makefile

```
@@ -3,12 +3,13 @@ NODE_VERSION ?=$(shell grep FROM Dockerfile | cut -d ':' -f 2| cut -d '-' -f 1)
 # We support two build modes , node-onbuild or alpine-node
 BUILD_TYPE ?=node-onbuild
 
-NAME_BUILD_CONTAINER ?=logentries-build-$(BUILD_TYPE)
-NAME_TEST_CONTAINER ?=logentries-test-$(BUILD_TYPE)
-NAME_EXPORT_CONTAINER ?=logentries-export-$(BUILD_TYPE)
+NAME_CONTAINER ?=r7insight_docker
```

Is this the name of the _image_ or the _container_?

---

Kind of used for both, more like the name of project to be prepended to image and container names, would you prefer this?
```suggestion
NAME_PROJECT ?=r7insight_docker
```

---

just `NAME`? :p

---

Sounds good